scripts/update_ambiguous_characters.py

```diff
@@ -25,22 +25,28 @@ def get_mapping_data() -> dict:
 def format_confusables_rs(raw_data: dict) -> str:
     """Format the downloaded data into a Rust source file."""
     prelude = """
-/// This file is auto-generated by scripts/update_ambiguous_characters.py.
+/// This file is auto-generated by `scripts/update_ambiguous_characters.py`.
 
 use once_cell::sync::Lazy;
 use rustc_hash::FxHashMap;
 
 /// Via: <https://github.com/hediet/vscode-unicode-data/blob/main/out/ambiguous.json>
 /// See: <https://github.com/microsoft/vscode/blob/095ddabc52b82498ee7f718a34f9dd11d59099a8/src/vs/base/common/strings.ts#L1094>
-pub(crate) static CONFUSABLES: Lazy<FxHashMap<u32, u8>> = Lazy::new(|| {
+pub(crate) static CONFUSABLES: Lazy<FxHashMap<u32, u32>> = Lazy::new(|| {
```

Comment:
Nit: I wonder if we should instead change the type to `<char, char>`. Having the `char`s in place could make the table a bit more readable and avoids the `unwrap` calls.

Reply:
@MichaReiser - That seems reasonable to me, but will it increase the size of this map?